cattle/agent/event.py

```diff
@@ -28,15 +28,18 @@ def _get_event_suffix(agent_id):
         return ';agent=' + agent_id
 
 
-def _data(events, agent_id):
-    event = {}
+def _events_query_string(events, agent_id):
+    params = []
+    suffix = ''
+
     if agent_id is not None:
-        event['agentId'] = agent_id
+        params.append('agentid=%s' % agent_id)
```

Comment:
just to make sure: are params case insensitive, and agentid is equivalent agentId?